src/C_TO_LOGIC.py

```diff
@@ -98,7 +98,10 @@ def GET_CPP_INCLUDES_LIST():  # -> list[str]: needs typing library?
                     path_list += ["-I" + thing_path]
     # Also include src files in git root dir
     dir_path = REPO_ABS_DIR()
+
+    # TODO: Temporarily include both paths to avoid breakage while still refactoring
     path_list += ["-I" + dir_path + "/"]
+    path_list += ["-I" + dir_path + "/pipelinec/include/"]
```

Comment:
Nice finding this chunk of code - should be pretty close to working

And I can help test